mod/wikirate/spec/set/right/filter_search_spec.rb

```diff
@@ -78,38 +78,38 @@
              Card::Env.params[:tag] = "nonexisitingtag"
 
             html = Card[:claim].format.render_core
-            expect(html).to have_tag('div',:with=>{:id=>"#{Card[:claim].name}+#{Card[:filter_search].cardname.url_key}"}) do    
+            expect(html).to have_tag('div',:with=>{:id=>"#{Card[:claim].name}+#{Card[:filter_search].cardname.url_key}"}) do
               without_tag('div',:with=>{:id=>"whateverclaim"})
             end
           end
           it "uses non related company" do
              Card::Env.params[:company] = "Iamnoangel"
             html = Card[:claim].format.render_core
-            expect(html).to have_tag('div',:with=>{:id=>"#{Card[:claim].name}+#{Card[:filter_search].cardname.url_key}"}) do    
+            expect(html).to have_tag('div',:with=>{:id=>"#{Card[:claim].name}+#{Card[:filter_search].cardname.url_key}"}) do
               without_tag('div',:with=>{:id=>"whateverclaim"})
             end
           end
           it "uses non related topic" do
              Card::Env.params[:topic] = "Iamnodemon"
             html = Card[:claim].format.render_core
-            expect(html).to have_tag('div',:with=>{:id=>"#{Card[:claim].name}+#{Card[:filter_search].cardname.url_key}"}) do    
+            expect(html).to have_tag('div',:with=>{:id=>"#{Card[:claim].name}+#{Card[:filter_search].cardname.url_key}"}) do
```

Comment:
Line is too long. [124/80]<br>Use the new Ruby 1.9 hash syntax.<br>Inconsistent indentation detected.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.